app/models/lockstep/connection.rb

```diff
@@ -3,7 +3,7 @@ class Lockstep::Connection < Lockstep::ApiRecord
   self.id_ref = 'company_id'
   load_schema(Schema::Company)
 
-  enum company_type: %w[Customer Vendor Group Company]
+  enum company_type: ['Customer', 'Vendor', 'Group', 'Company', 'Third Party'].freeze
```

Comment:
@rishirajkumar97  why don't we have a company_type CustomerVendor here ? we do see it at Inbox end rite ?

Reply:
No we dont